### DISCUSSION TITLE
How to prioritize release candidates over snapshot versions?

### DISCUSSION THREAD

**Discussion by 0utplay** [renovatebot/renovate#31880] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi, I am currently running the following configuration of renovate https://github.com/CloudNetService/CloudNet/blob/nightly/.github/renovate.json

I would like renovate to keep release candidate versions instead of switching to snapshots.
![image](https://github.com/user-attachments/assets/9aecce8e-19bd-4f78-b34a-e29d5958544c)
As you can see here renovate is trying to update from 1.0.0-rc.1 to 1.0.0-SNAPSHOT which I do not really like as the RC is the more stable version here.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
As far as I'm aware, this is how Maven sorts versions too. If you don't want normal sorting, you'll need to write a package rule which excludes snapshots using `allowedVersions`
